tests/exchanges/traders/test_trader.py

```diff
@@ -557,7 +557,8 @@ async def test_close_filled_sell_limit_order_one_in_two(self):
         assert second_limit_buy in orders_manager.get_open_orders()
 
         assert initial_portfolio != portfolio_manager.portfolio
-        assert portfolio_manager.portfolio.portfolio["BQX"][commons_constants.PORTFOLIO_AVAILABLE] == 2
+        # 0.002 as fees
+        assert portfolio_manager.portfolio.portfolio["BQX"][commons_constants.PORTFOLIO_AVAILABLE] == 1.998
```

Comment:
What changed ?

Reply:
the way binance fees are computed in ccxt

Reply:
I don't understand why this is the only test impacted. If ccxt is now computing fees it's a major change

Reply:
due to this deletion, the way ccxt computes fees for binance has changed https://github.com/ccxt/ccxt/commit/bbf545f84eee84fc930b8c89fe140cac03664531#diff-2fdc97a66b336a92bf28e3ea3665811efbb6c89e3d3c5047ad5265d483188a0bL1063
This is the only test which relies on ccxt fees apparently